src/Discovery.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Http\Discovery;
+
+interface Discovery
```

Comment:
Anyway, the idea of communicating that they are something similar is good. But interface cannot be really used here.